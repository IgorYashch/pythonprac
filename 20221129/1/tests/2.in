class C(metaclass=dump):
    x = 345

    foo = lambda self: print('Hello, world!', C.x)

    def print_digits(self, *digits, alpha=0, beta=10):
        print(*digits, alpha, beta, sep=', ')

c = C()
c.foo()
c.print_digits(12,14,56,34,23, beta=34754385)
