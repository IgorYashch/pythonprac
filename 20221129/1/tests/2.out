foo: (), {}
Hello, world! 345
print_digits: (12, 14, 56, 34, 23), {'beta': 34754385}
12, 14, 56, 34, 23, 0, 34754385
